easybuild/easyconfigs/d/DualSPHysics/DualSPHysics-5.0.175-GCC-11.2.0-CUDA-11.4.1.eb

```diff
@@ -18,6 +18,7 @@ description = """DualSPHysics is based on the Smoothed Particle Hydrodynamics mo
  with real-life engineering problems."""
 
 toolchain = {'name': 'GCC', 'version': '11.2.0'}
+toolchainopts = {'rpath': False}
```

Comment:
this is needed? From what i can tell form the easyblock, there is some very explicit code that supposedly handles rpath, so i could expect it to be supported. And if not, should also the non-CUDA version revert to rpath False?

@jfgrimm you wrote this easyblock, please let me know :)

Reply:
closing for #23382 / https://github.com/easybuilders/easybuild-easyblocks/pull/3835